WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -107,9 +108,18 @@ protected override async Task ActionAsync(CancellationToken cancel)
 
 			// RoundStates have to contain all states. Do not change stateId=0.
 			SetRoundStates();
+
+			RequestTimeStatista.Instance.Add("arena-period-inside", DateTimeOffset.UtcNow - beforeInside);
```

Comment:
What are we gonna achieve with this? How much time does it take to acquire the lock and let it go?

Reply:
We don't know for sure. We will measure it. It depends on whether someone else is holding the lock. Also, we make sure the cause of the lagg is the lock and not something inside.